chainercv/datasets/__init__.py

```diff
@@ -5,6 +5,8 @@
 from chainercv.datasets.cub.cub_keypoint_dataset import CUBKeypointDataset  # NOQA
 from chainercv.datasets.cub.cub_label_dataset import CUBLabelDataset  # NOQA
 from chainercv.datasets.cub.cub_utils import cub_label_names  # NOQA
+from chainercv.datasets.directory_parsing_classification_dataset import DirectoryParsingClassificationDataset  # NOQA
+from chainercv.datasets.directory_parsing_classification_dataset import parse_label_names  # NOQA
```

Comment:
`parse_label_names` is too ambiguous. If you want to import it under `chainercv.datasets`, the name should be more specific. How about `directory_parsing_label_names`? I think the name should be similar to `DirectoryParsingClassificationDataset` in order to show the relationship.